### PR TITLE
Mark/hit logout api

### PR DESCRIPTION
**Pull request by railsadvocate**
- Hit logout API when users logout
- Add API_VERSION variable
- fix code spacing issues

### CODE REVIEW

File: app/src/main/java/edu/wwu/avilatstudents/journey/MainActivity.java [hunk 1]
```diff
@@ -182,14 +186,18 @@ public void onClick(View view) {
...
                 actionBar.show();
                 // reset nav buttons to base color (selected color applied appropriately in switch)
                 setNavButtonColorSelected(journeysFab);
                 updateActionBar(R.id.journeys_fab);
                 visibleLayout.setVisibility(View.GONE);
                 journeysLayout.setVisibility(View.VISIBLE);
                 visibleLayout = journeysLayout;
-                visibleLayout.requestFocus();
+
+                Intent gotoLogin = new Intent(MainActivity.this, LoginActivity.class);
```
Comment (avila000) [inline app/src/main/java/edu/wwu/avilatstudents/journey/MainActivity.java]: sessionManager.logout() starts the Login Activity already, so there is no need for this.

Comment (railsadvocate) [reply app/src/main/java/edu/wwu/avilatstudents/journey/MainActivity.java]: woops, yeah I meant to get rid of that haha. One sec

Comment (railsadvocate) [reply app/src/main/java/edu/wwu/avilatstudents/journey/MainActivity.java]: fixed

File: app/src/main/java/edu/wwu/avilatstudents/journey/MainActivity.java [hunk 2]
```diff
@@ -34,6 +34,8 @@
 
     private String username;
 
+    private DatabaseManager dbm;
```
Comment (avila000) [inline app/src/main/java/edu/wwu/avilatstudents/journey/MainActivity.java]: There is a Databasemanager object already created in OnCreate() We can use whichever object you think is best, but we should only have one

Comment (railsadvocate) [reply app/src/main/java/edu/wwu/avilatstudents/journey/MainActivity.java]: I didn't see that. I'll delete what I added and just use the one you defined.

**Review by avila000 — APPROVED** [review]

**Pull request merged**